packages/inline-choice/controller/src/index.js

```diff
@@ -4,64 +4,64 @@ export function outcome(model, session) {
   return new Promise((resolve) => {
     resolve({});
   });
-
 }
 
-export function model(model, session, env) {
+const colorMap = {
+  black_on_rose: 'black-on-rose',
+  white_on_black: 'white-on-black',
+  black_on_white: 'default'
+};
 
-  const colorMap = {
-    black_on_rose: 'black-on-rose',
-    white_on_black: 'white-on-black',
-    black_on_white: 'default'
-  };
+const findLabel = (locale, langs) => (langs || []).find(l => l.lang === locale);
 
-  function filterItemsArrByLocale(locale, items) {
-    if (!items) {
-      return [];
-    }
-    return items.filter(item => locale === item.lang);
-  }
+const prepChoices = (locale, choicesArr) => choicesArr.map(choice => ({
+  value: choice.value,
+  label: (findLabel(locale, choice.label) || {}).value,
+}));
 
-  function filterQuestionChoicesByLocale(locale, choicesArr) {
-
-    return choicesArr.map((choice) => {
-        let choiceObj = {};
-        choiceObj["value"] = choice.value;
-        choiceObj["correct"] = choice.correct || false;
-        choiceObj["label"] = filterItemsArrByLocale(locale,choice.label)
```

Comment:
don't return `[{lang:string, value:string}]` return `string` - this makes it alot easier for the component to do it's work.